docs/reference/commandline/swarm_join.md

```diff
@@ -17,52 +17,52 @@ Usage:  docker swarm join [OPTIONS] HOST:PORT
 Join a Swarm as a node and/or manager
 
 Options:
-      --ca-hash string      Hash of the Root Certificate Authority certificate used for trusted join
       --help                Print usage
       --listen-addr value   Listen address (default 0.0.0.0:2377)
-      --manager             Try joining as a manager.
-      --secret string       Secret for node acceptance
+      --token string        Token for entry into the swarm
 ```
 
-Join a node to a Swarm cluster. If the `--manager` flag is specified, the docker engine
-targeted by this command becomes a `manager`. If it is not specified, it becomes a `worker`.
+Join a node to a Swarm cluster. If the token given to the `--token` flag is a manager token,
```

Comment:
I'm afraid `Swarm` isn't consistently lowercased. The "swarm join" reference page uses a mix of title case and lower case, and messages from the CLI itself tend to use title case; for example "This node joined a Swarm as a worker.".

Should I go through and change these?